gateway-modules/apim-ballerina-configs/configs/org/wso2/carbon/apimgt/gateway/utils/APICoreUtil.bal

```diff
@@ -61,7 +73,7 @@ function getAPIs () (json) {
     json apiList;
     try {
         http:ClientConnector client = create http:ClientConnector(getAPICoreURL());
-        string query = "?labels=Private,Public";
+        string query = "?labels=Default";
```

Comment:
This label value we have to read from a config file or an environment variable.  I am not sure whether the env support is there in Ballerina at the moment. We should keep in mind that this value can be changed.